src/bindings.h

```diff
@@ -111,6 +107,8 @@ struct lxcfs_opts {
 	bool swap_off;
 	bool use_pidfd;
 	bool use_cfs;
+        // As of opts version 2.
```

Comment:
we need to place all the new fields at the end of the struct

Reply:
Updated